project_organization.md

```diff
@@ -0,0 +1,57 @@
+## Industrial Advisory Board
+
```

Comment:
These first two sub-sections are repeated below - I think perhaps these initial sub-sections should be removed?